helios/view_utils.py

```diff
@@ -57,10 +57,10 @@ def prepare_vars(request, vars):
             from helios.models import Trustee
             trustee = Trustee.objects.get(uuid=session.get('helios_trustee_uuid'))
             election = trustee.election
-        except:
+        except Trustee.DoesNotExist:
             try:
                 del session['helios_trustee_uuid']
-            except:
+            except NameError:
```

Comment:
See comment before, I think it'd be better to just check `if 'helios_trustee_uuid'` in session instead of catching exception.